index.html

```diff
@@ -227,7 +233,7 @@ <h3 class="subtitle is-5">We don't hide what we do</h3>
           <div class="column is-4">
             <article class="card is-shady">
               <div class="card-image has-text-centered">
-                <i class="fa fa-paw" role="img" alt="paw logo" aria-label="paw logo"></i>
+                <i class="fa fa-paw" role="img" alt="paw logo" aria-label="paw logo" aria-hidden="true"></i>
```

Comment:
Remove role and alt, aria-hidden is enough and correct. Tag i hasn't alt attribute, that is incorrect